pkg/ebpftracer/c/headers/vmlinux_missing.h

```diff
@@ -398,4 +404,66 @@ enum {
 	BPF_SK_STORAGE_GET_F_CREATE  = BPF_LOCAL_STORAGE_GET_F_CREATE,
 };
 
+/*
+ * Copy of sock_ops operations.
+ * See: https://elixir.bootlin.com/linux/latest/source/include/uapi/linux/bpf.h#L6233.
+ */
+enum {
+	BPF_SOCK_OPS_VOID                   = 0,
+	BPF_SOCK_OPS_TIMEOUT_INIT           = 1,
+	BPF_SOCK_OPS_RWND_INIT              = 2,
+	BPF_SOCK_OPS_TCP_CONNECT_CB         = 3,
+	BPF_SOCK_OPS_ACTIVE_ESTABLISHED_CB  = 4,
+	BPF_SOCK_OPS_PASSIVE_ESTABLISHED_CB = 5,
+	BPF_SOCK_OPS_NEEDS_ECN              = 6,
+	BPF_SOCK_OPS_BASE_RTT               = 7,
+	BPF_SOCK_OPS_RTO_CB                 = 8,
+	BPF_SOCK_OPS_RETRANS_CB             = 9,
+	BPF_SOCK_OPS_STATE_CB               = 10,
+	BPF_SOCK_OPS_TCP_LISTEN_CB          = 11,
+	BPF_SOCK_OPS_RTT_CB                 = 12,
+	BPF_SOCK_OPS_PARSE_HDR_OPT_CB       = 13,
+	BPF_SOCK_OPS_HDR_OPT_LEN_CB         = 14,
+	BPF_SOCK_OPS_WRITE_HDR_OPT_CB       = 15,
+};
+
+/*
+ * Copy of definitions for bpf_sock_ops_cb_flags.
+ * See: https://elixir.bootlin.com/linux/latest/source/include/uapi/linux/bpf.h#L6178.
+ */
+enum {
+	BPF_SOCK_OPS_RTO_CB_FLAG                   = 1,
+	BPF_SOCK_OPS_RETRANS_CB_FLAG               = 2,
+	BPF_SOCK_OPS_STATE_CB_FLAG                 = 4,
+	BPF_SOCK_OPS_RTT_CB_FLAG                   = 8,
+	BPF_SOCK_OPS_PARSE_ALL_HDR_OPT_CB_FLAG     = 16,
+	BPF_SOCK_OPS_PARSE_UNKNOWN_HDR_OPT_CB_FLAG = 32,
+	BPF_SOCK_OPS_WRITE_HDR_OPT_CB_FLAG         = 64,
+	BPF_SOCK_OPS_ALL_CB_FLAGS                  = 127,
+};
+
+/*
+ * Copy of bpf.h's bpf_sock_ops with minimal subset
+ * of fields used by the tcprtt_sockops example.
+ * See: https://elixir.bootlin.com/linux/latest/source/include/uapi/linux/bpf.h#L6101.
+ */
+struct bpf_sock_ops {
+	__u32 op;
+	union {
+		__u32 args[4];
+		__u32 reply;
+		__u32 replylong[4];
+	};
+	__u32 family;
+	__u32 remote_ip4;	/* Stored in network byte order */
+    __u32 local_ip4;	/* Stored in network byte order */
+    __u32 remote_ip6[4];/* Stored in network byte order */
+    __u32 local_ip6[4];	/* Stored in network byte order */
+    __u32 remote_port;	/* Stored in network byte order */
+    __u32 local_port;	/* stored in host byte order */
+	__u32 srtt_us;
+    __u32 bpf_sock_ops_cb_flags;
+    __bpf_md_ptr(struct bpf_sock *, sk);
```

Comment:
There is no need for this tbh. I would go with the exact definition that is spit out by `bpftool btf dump file /sys/kernel/btf/vmlinux format c`.